utils/frame/frame-utilities-cli/src/inspect.rs

```diff
@@ -0,0 +1,89 @@
+// This file is part of Substrate.
```

Comment:
why is this command here and not in `sc-cli`? This isn't FRAME specific.

Reply:
it's part of the key subcommand. and one of them (insert_key) is frame-specific.